app/_kong_plugins/graphql-rate-limiting-advanced/index.md

```diff
@@ -33,12 +33,232 @@ categories:
+
+This is different from the cost strategy ([`config.cost_strategy`](/plugins/graphql-rate-limiting-advanced/reference/#schema--config-cost_strategy)), which is the method that the plugin uses to determine GraphQL query costs.
+
+{:.warning}
+> A `cluster` strategy with a sync rate of `-1` should not be used in production with hybrid mode, DB-less mode, or Konnect, as it creates security risks.
+
+## Introspection endpoint
+
+The introspection endpoint is generated based on the [Gateway Service path](/gateway/entities/service/), so you must define a path in the Gateway Service itself, 
+instead of appending from the Route path.
+  
+The query and introspection endpoints _cannot_ have separate paths.
+
+For example, when using KIC, if the query and introspection endpoints are at the path `/graphql`, they should be configured like this:
+* Add the `konghq.com/strip-path: "true"` annotation to the Ingress resource
+* Add the `konghq.com/path: /graphql` annotation to the Service resource
+
+## Managing costs in GraphQL queries
+
+{{site.base_gateway}} evaluates GraphQL query costs by introspecting the endpoint's GraphQL schema and applying cost decoration to parts of the schema tree.
+
+Initially all nodes start with zero cost, with any operation at cost 1.
+You can add rate limiting constraints on any subtree. 
+If a subtree is omitted, then the rate limit window applies to the whole tree, meaning any operation.
+
+The GraphQL Rate Limiting Advanced plugin exposes two strategies for approximating the cost of a GraphQL query, configurable via [`config.cost_strategy`](/plugins/graphql-rate-limiting-advanced/reference/#schema--config-cost_strategy):
+* [`default`](#default-strategy): The default strategy tries to estimate cost on queries by counting the nesting of nodes. 
+* [`node_quantifier`](#node_quantifier-strategy): Useful for GraphQL schemas that enforce quantifier arguments on any connection.
+
+### default strategy
+
+The default strategy is meant as a good middle ground for general GraphQL
+queries, where it's difficult to assert a clear cost strategy, so every operation
+has a cost of 1.
+
+For example:
+
+```
+query { # + 1
+  allPeople {  # + 1
+    people { # + 1
+      name # + 1
+    }
+  }
+}
+# total cost: 4
+```
+
+Default node costs can be defined by decorating the schema:
+
+| `type_path`                | `mul_arguments`   | `mul_constant`    | `add_arguments`   | `add_constant` |
+|----------------------------|-------------------|-------------------|-------------------|----------------|
+| `Query.allPeople`          | ["first"]         | 1                 | []                | 1              |
+| `Person.vehicleConnection` | ["first"]         | 1                 | []                | 1              |
+
+
```

Comment:
```suggestion

In this example, `vehicleConnection` weight (4) is applied 10 times, and the total weight of it (40) 20 times, which gives us a rough 800:
```
I wanted a bit of text before the example to help contextualize it, so I moved this here.